src/utils/pricing.ts

```diff
@@ -13,6 +13,15 @@ export const KSM = '2124-2-256'
 export const WHITELIST: string[] = [
     '2124-0-0', // wnative
```

Comment:
Should we also whitelist the native token on Pendulum ie `2094-0-0`? The check in `findEthPerToken()` should then probably check whether it's the native token on Pendulum or Amplitude and then return `ONE_BD`.

Reply:
Yes, good catch. Otherwise it will probably fail with Pendulum. I will also modify for wherever `WNATIVE` is used so it takes the proper constant.

Reply:
Thanks for refactoring it into constants 👍 

I thought about the whole price setup again and realized that we need to handle the prices a little differently on Pendulum vs Amplitude. The reason is that the PEN token is listed on a CEX and thus the price is directly provided from the DIA price oracle. The AMPE token is not listed on a CEX so we have to derive the price from the AMPE/KSM Zenlink pool. I committed a small fix for this, please tell me what you think @gianfra-t

Reply:
It totally makes sense. Didn't think about getting it from the oracle !